docs/source/advanced/backends.rst

```diff
@@ -434,3 +528,121 @@ The following tables list the functions available in the `CUDA Runtime API <http
 **Version Management**
 
 *not available*
+
+
+
+HIP
+```
+
+Current restrictions on HCC platform
+++++++++++++++++++++++++++++++++++++
```

Comment:
We need to decide whether such titles have every word (except prepositions and articles) capitalized or not. Now it's a mix in this file. This could be done later as well

Reply:
I did it for the whole doc. Was not much work.